src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -36,6 +40,9 @@
 	@Option( names = "--all", description = "Accept all differences from the given test report." )
 	private boolean all;
 
+	@Option( names = "--exclude", description = "Ignore matching elements during accept." )
```

Comment:
Is `--exclude` fixe? Maybe stick to `--ignore` or so? CC @roesslerj

Reply:
Description suggestion: "Filter(s) to exclude changes from the commit." (And maybe adapt "exclude" if we choose another param name.)